tensorflow_addons/custom_ops/activations/cc/kernels/gelu_op.h

```diff
@@ -137,8 +138,7 @@ void GeluGradOp<Device, T>::OperateNoTemplate(OpKernelContext* context,
           approximate, output->flat<T>());
 }
 
-}  // namespace tensorflow
+}  // end namespace addons
+}  // namespace tensorflow#undef EIGEN_USE_THREADS
 
-#undef EIGEN_USE_THREADS
```

Comment:
why remove #undef EIGEN_USE_THREADS

Reply:
@tomerk When you get a moment could you confirm if this was or was not a mistake? After this issue we should be ready to merge.

Reply:
Whoops, this should be fixed now!